sigs/sig-cnc-charter.md

```diff
@@ -0,0 +1,60 @@
+* There is not a fixed number of Maintainer and Trainer positions available; they are filled based on demonstrated skill, merit, enthusiasm, and member demand for the machines.
+* When changing leadership, responsibilities should be officially transitioned to the new party on the first day of the month following an election for new leadership, or as privately discussed between the previous and incoming leadership.
+
+## Scope of the CNC SIG
+
+The CNC SIG exists to facilitate safe and effective usage of the CNC routing equipment at denhac. This group will maintain CNC equipment, respond to future events and large scale projects proposed by the membership, as well as oversee additions and changes to the CNC routing space and equipment within denhac.
+
+#### ***In-Scope***
+
+* Maintaining denhac-owned CNC routers and associated hardware including the computing infrastructure required to operate the machines
+* Resources for use and maintenance specific to the CNC routers such as workholding, calibration tools, etc
+* Project support and collaboration with other sigs and future denhac events
+* Maintenance and Improvements to the CNC routers (Shapeoko 5 Pro, Space Owned CNC Mill/Routers) and related space within denhac
+* Responding to and managing the use of donated CNC routers and equipment
+* Responding to and managing the use of member-owned CNC routing equipment on loan to denhac
+* Addressing membership concerns and questions specific to equipment within the CNC router area
+* Disciplinary measures such as removal of CNC authorization or related privileges
+* Managing the spending and allocation of SIG funds, as well as coordinating Board Requests for additional funds
+* Creating and teaching a comprehensive curriculum regarding safe and effective usage of the CNC routers to all interested and permitted members
+* Defining the required knowledge needed for Authorized Trainers and Maintainers, as well as providing this knowledge to certify denhac members as Trainers and Maintainers of the CNC routers when appropriate
+* Defining policy regarding specific materials and end effectors that are approved for use with the CNC routers at denhac
+
+#### ***Out-of-Scope***
+
+* Maintenance and upkeep of member-owned CNC machines that aren't for use by denhac members
+* Member-owned materials
+* Anything not specifically listed as In-Scope, or as determined by the SIG Leads
+
+## Organization Management
+
+SIG decisions will officially be made by majority vote between the SIG leads who should strongly take into consideration the sentiments of any denhac member with a special interest in the CNC routers on site, while ultimately acting in the best interests of denhac as a whole. SIG leads are encouraged to make decisions using majority votes from CNC SIG members. Decisions made by the CNC SIG leads can always be overridden by the denhac board of directors.
```

Comment:
could use rewording at "SIG decisions will officially be made by majority vote between the SIG leads" - there are 2 leads, what's a majority vote between 2 people?

suggest reword to: SIG decisions will be made by the SIG leads, who should strongly take into consideration...